test/functional/feature_onlynet.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Test -onlynet
+"""
+
+from test_framework.basic_server import (
+    BasicServer,
+    tor_control
+)
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+)
+
+
+class OnlynetTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.extra_args = [
```

Comment:
I needed this line for the test to run:
```diff
 class OnlynetTest(BitcoinTestFramework):
     def set_test_params(self):
+        self.setup_clean_chain = True
         self.extra_args = [
```
Otherwise, was seeing the following error:

<details><summary>test output</summary><p>

```
$ test/functional/feature_onlynet.py 
2021-08-11T09:22:40.305000Z TestFramework (INFO): Initializing test directory /tmp/bitcoin_func_test_p1tdzv8m
2021-08-11T09:22:41.628000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/home/jon/projects/bitcoin/bitcoin/test/functional/test_framework/test_framework.py", line 130, in main
    self.setup()
  File "/home/jon/projects/bitcoin/bitcoin/test/functional/test_framework/test_framework.py", line 285, in setup
    self.setup_network()
  File "/home/jon/projects/bitcoin/bitcoin/test/functional/test_framework/test_framework.py", line 379, in setup_network
    self.setup_nodes()
  File "/home/jon/projects/bitcoin/bitcoin/test/functional/test_framework/test_framework.py", line 409, in setup_nodes
    assert_equal(n.getblockchaininfo()["blocks"], 199)
  File "/home/jon/projects/bitcoin/bitcoin/test/functional/test_framework/util.py", line 49, in assert_equal
    raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
AssertionError: not(0 == 199)
2021-08-11T09:22:41.680000Z TestFramework (INFO): Stopping nodes
2021-08-11T09:22:41.860000Z TestFramework (WARNING): Not cleaning up dir /tmp/bitcoin_func_test_p1tdzv8m
2021-08-11T09:22:41.860000Z TestFramework (ERROR): Test failed. Test logging available at /tmp/bitcoin_func_test_p1tdzv8m/test_framework.log
2021-08-11T09:22:41.861000Z TestFramework (ERROR): 
2021-08-11T09:22:41.861000Z TestFramework (ERROR): Hint: Call /home/jon/projects/bitcoin/bitcoin/test/functional/combine_logs.py '/tmp/bitcoin_func_test_p1tdzv8m' to consolidate all logs
2021-08-11T09:22:41.862000Z TestFramework (ERROR): 
2021-08-11T09:22:41.862000Z TestFramework (ERROR): If this failure happened unexpectedly or intermittently, please file a bug and provide a link or upload of the combined log.
2021-08-11T09:22:41.862000Z TestFramework (ERROR): https://github.com/bitcoin/bitcoin/issues
2021-08-11T09:22:41.862000Z TestFramework (ERROR): 
```
</p></details>

Reply:
Hm, after coming back here and rebuilding, the test now runs for me without it. Strange.  Sorry for the noise.